woocommerce/woocommerce-ios
# Disable user interaction on App Logs text view

Fixes #1275 

This PR disables user interaction in the App Log textview. The "share" icon in the nav bar is the only way the user should be able to interact with the text found in the App Log.

## To Test
1. Navigate to My store > Settings (gear icon) > Help & Support > View Application Log > Current (or any date stamp). 
2. Try to long press to select the text
3. Try to tap the text to select 
4. Note that no keyboard appears.

Update release notes:
